lib/dish.rb

```diff
@@ -0,0 +1,15 @@
+class Dish
+  attr_reader :description, :price
+  def initialize(description: description, price: price)
+    @description = description
+    @price = price
+  end
+
+  def update_quantity(quantity)
+    raise "Quantity is negative" if quantity < 0
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)